### ISSUE TITLE
Do you have a plan to support the Scratch feature.

### ISSUE THREAD

**Issue by joowh85** [moxley/atom-ruby-test#34]
Do you have a plan to support the Scratch feature.
It's comfortable to use.

See below

https://www.facebook.com/photo.php?fbid=763377550420698&set=a.206943152730810.49635.100002453811057&type=1

**Comment by moxley** [comment]
Hi @joowh85. Your link appears to be broken. Can you give an explanation of the feature?

**Comment by joowh85** [comment]
I think that link is my Facebook account, so you don't have permission for read my article.

So I draw below (: 

This is the Atom window.

---------------------------------------------
   File     Edit    View     Help...
---------------------------------------------
                            |
Source Code       |      Ruby test
                            |
---------------------------------------------

**Comment by moxley** [comment]
No, I don't have any plans to support opening up the test pane on the right side, as it isn't helpful for me. Other people are free to submit a pull request to support this feature.

**Comment by madis** [comment]
Since this issue is open, I thought posting here instead of creating a new one.

I'm having similar issue - working on a 40" 4k monitor, using splitting panes actively. When running atom-ruby test, output bottom panel takes whole width of the screen.

I tried [atom-rspec](https://github.com/fcoury/atom-rspec) and it has better approach for me: uses *ScrollView*, opening it in another tab, allowing user to put it anywhere.

My question: would @moxley be interested in incorporating implementation of this approach to his *atom-ruby-test* package if I implemented it?

**Comment by nickcoyne** [comment]
+1 for this feature.

**Comment by vanboom** [comment]
+1 also - just purchased a wide screen monitor and wanting to have a right hand panel for running tests
